gensim/models/word2vec.py

```diff
@@ -478,7 +480,9 @@ def __init__(self, sentences=None, corpus_file=None, size=100, alpha=0.025, wind
                  max_vocab_size=None, sample=1e-3, seed=1, workers=3, min_alpha=0.0001,
                  sg=0, hs=0, negative=5, ns_exponent=0.75, cbow_mean=1, hashfxn=hash, iter=5, null_word=0,
                  trim_rule=None, sorted_vocab=1, batch_words=MAX_WORDS_IN_BATCH, compute_loss=False, callbacks=(),
-                 max_final_vocab=None):
+                 max_final_vocab=None,
+                 pretrained_emb=None 
```

Comment:
```suggestion
                 pretrained_emb=None,
```

Reply:
No, "pretrained_emb=None" is the last param, there should not ,